docs/components/store.md

```diff
@@ -296,9 +296,15 @@ Currently only memcached "backend" is supported:
 ```yaml
 type: MEMCACHED # Case-insensitive
 config:
-  addresses:
-    - localhost:11211
-
+  addresses: []
+  timeout: 0s
+  max_idle_connections: 0
+  max_async_concurrency: 0
```

Comment:
What if we would actually write down the default values here and in the other place in this file? I am talking about this: https://github.com/thanos-io/thanos/blob/1fff9a7157d1f3b7d5e0ce36f999b10bb3ae7ec4/pkg/cacheutil/memcached_client.go#L46-L55. I think it would be even more convenient for Thanos users.

Reply:
Seems logical to do. Currently I lack some time, but will adjust it soon